<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>CampainLeaderboardWindow</class>
 <widget class="QWidget" name="CampainLeaderboardWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>1000</width>
    <height>700</height>
   </rect>
  </property>
  <property name="minimumSize">
   <size>
    <width>1000</width>
    <height>700</height>
   </size>
  </property>
  <property name="maximumSize">
   <size>
    <width>1000</width>
    <height>700</height>
   </size>
  </property>
  <property name="windowTitle">
   <string>camPAIN Leaderboard</string>
  </property>
  <property name="styleSheet">
   <string notr="true">     background-color:white;
</string>
  </property>
  <widget class="QPushButton" name="returnToMenuButton">
   <property name="geometry">
    <rect>
     <x>20</x>
     <y>20</y>
     <width>125</width>
     <height>40</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <pointsize>17</pointsize>
    </font>
   </property>
   <property name="styleSheet">
    <string notr="true">QPushButton
{
	color:red;
	border: 1px solid #000000;
					 border-radius: 15px;
}

QPushButton:pressed
{
	color:red;
     background-color:rgba(0, 145, 255, 128);
}
</string>
   </property>
   <property name="text">
    <string>Game Menu</string>
   </property>
  </widget>
  <widget class="QPushButton" name="globalLeaderboardButton">
   <property name="geometry">
    <rect>
     <x>370</x>
     <y>180</y>
     <width>113</width>
     <height>32</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <pointsize>17</pointsize>
    </font>
   </property>
   <property name="styleSheet">
    <string notr="true">QPushButton
{
	color:blue;
	border: 1px solid #000000;
					 border-radius: 15px;
}

QPushButton:pressed
{
	color:blue;
     background-color:rgba(0, 145, 255, 128);
}
</string>
   </property>
   <property name="text">
    <string>Global</string>
   </property>
  </widget>
  <widget class="QPushButton" name="pushButton_2">
   <property name="geometry">
    <rect>
     <x>500</x>
     <y>180</y>
     <width>113</width>
     <height>32</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <pointsize>17</pointsize>
    </font>
   </property>
   <property name="styleSheet">
    <string notr="true">QPushButton
{
	color:blue;
	border: 1px solid #000000;
					 border-radius: 15px;
}

QPushButton:pressed
{
	color:blue;
     background-color:rgba(0, 145, 255, 128);
}
</string>
   </property>
   <property name="text">
    <string>Group</string>
   </property>
  </widget>
  <widget class="QGroupBox" name="groupBox">
   <property name="geometry">
    <rect>
     <x>150</x>
     <y>230</y>
     <width>700</width>
     <height>400</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <pointsize>20</pointsize>
    </font>
   </property>
   <property name="styleSheet">
    <string notr="true">color:red;
	border: 1px solid #000000;
					 border-radius: 15px;
</string>
   </property>
   <property name="title">
    <string>Global Leaderboard</string>
   </property>
   <widget class="QLabel" name="scoreLabel5">
    <property name="geometry">
     <rect>
      <x>450</x>
      <y>190</y>
      <width>175</width>
      <height>25</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <pointsize>20</pointsize>
     </font>
    </property>
    <property name="styleSheet">
     <string notr="true">	border: 0px solid #000000;
</string>
    </property>
    <property name="text">
     <string/>
    </property>
   </widget>
   <widget class="QLabel" name="label_6">
    <property name="geometry">
     <rect>
      <x>50</x>
      <y>160</y>
      <width>21</width>
      <height>25</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <pointsize>20</pointsize>
     </font>
    </property>
    <property name="styleSheet">
     <string notr="true">	border: 0px solid #000000;
</string>
    </property>
    <property name="text">
     <string>4.</string>
    </property>
   </widget>
   <widget class="QLabel" name="usernameLabel4">
    <property name="geometry">
     <rect>
      <x>125</x>
      <y>160</y>
      <width>250</width>
      <height>25</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <pointsize>20</pointsize>
     </font>
    </property>
    <property name="styleSheet">
     <string notr="true">	border: 0px solid #000000;
</string>
    </property>
    <property name="text">
     <string/>
    </property>
   </widget>
   <widget class="QLabel" name="usernameLabel6">
    <property name="geometry">
     <rect>
      <x>125</x>
      <y>220</y>
      <width>250</width>
      <height>25</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <pointsize>20</pointsize>
     </font>
    </property>
    <property name="styleSheet">
     <string notr="true">	border: 0px solid #000000;
</string>
    </property>
    <property name="text">
     <string/>
    </property>
   </widget>
   <widget class="QLabel" name="scoreLabel9">
    <property name="geometry">
     <rect>
      <x>450</x>
      <y>310</y>
      <width>175</width>
      <height>25</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <pointsize>20</pointsize>
     </font>
    </property>
    <property name="styleSheet">
     <string notr="true">	border: 0px solid #000000;
</string>
    </property>
    <property name="text">
     <string/>
    </property>
   </widget>
   <widget class="QLabel" name="usernameLabel8">
    <property name="geometry">
     <rect>
      <x>125</x>
      <y>280</y>
      <width>250</width>
      <height>25</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <pointsize>20</pointsize>
     </font>
    </property>
    <property name="styleSheet">
     <string notr="true">	border: 0px solid #000000;
</string>
    </property>
    <property name="text">
     <string/>
    </property>
   </widget>
   <widget class="QLabel" name="label_9">
    <property name="geometry">
     <rect>
      <x>50</x>
      <y>250</y>
      <width>21</width>
      <height>25</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <pointsize>20</pointsize>
     </font>
    </property>
    <property name="styleSheet">
     <string notr="true">	border: 0px solid #000000;

</string>
    </property>
    <property name="text">
     <string>7.</string>
    </property>
   </widget>
   <widget class="QLabel" name="label_2">
    <property name="geometry">
     <rect>
      <x>450</x>
      <y>30</y>
      <width>175</width>
      <height>31</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <pointsize>20</pointsize>
     </font>
    </property>
    <property name="styleSheet">
     <string notr="true">	border: 0px solid #000000;
	color:blue;</string>
    </property>
    <property name="text">
     <string>Score</string>
    </property>
   </widget>
   <widget class="QLabel" name="scoreLabel4">
    <property name="geometry">
     <rect>
      <x>450</x>
      <y>160</y>
      <width>175</width>
      <height>25</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <pointsize>20</pointsize>
     </font>
    </property>
    <property name="styleSheet">
     <string notr="true">	border: 0px solid #000000;
</string>
    </property>
    <property name="text">
     <string/>
    </property>
   </widget>
   <widget class="QLabel" name="usernameLabel3">
    <property name="geometry">
     <rect>
      <x>125</x>
      <y>130</y>
      <width>250</width>
      <height>25</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <pointsize>20</pointsize>
     </font>
    </property>
    <property name="styleSheet">
     <string notr="true">	border: 0px solid #000000;
</string>
    </property>
    <property name="text">
     <string/>
    </property>
   </widget>
   <widget class="QLabel" name="scoreLabel10">
    <property name="geometry">
     <rect>
      <x>450</x>
      <y>340</y>
      <width>175</width>
      <height>25</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <pointsize>20</pointsize>
     </font>
    </property>
    <property name="styleSheet">
     <string notr="true">	border: 0px solid #000000;
</string>
    </property>
    <property name="text">
     <string/>
    </property>
   </widget>
   <widget class="QLabel" name="label_4">
    <property name="geometry">
     <rect>
      <x>50</x>
      <y>100</y>
      <width>21</width>
      <height>25</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <pointsize>20</pointsize>
     </font>
    </property>
    <property name="styleSheet">
     <string notr="true">	border: 0px solid #000000;
</string>
    </property>
    <property name="text">
     <string>2.</string>
    </property>
   </widget>
   <widget class="QLabel" name="scoreLabel7">
    <property name="geometry">
     <rect>
      <x>450</x>
      <y>250</y>
      <width>175</width>
      <height>25</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <pointsize>20</pointsize>
     </font>
    </property>
    <property name="styleSheet">
     <string notr="true">	border: 0px solid #000000;
</string>
    </property>
    <property name="text">
     <string/>
    </property>
   </widget>
   <widget class="QLabel" name="scoreLabel2">
    <property name="geometry">
     <rect>
      <x>450</x>
      <y>100</y>
      <width>175</width>
      <height>25</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <pointsize>20</pointsize>
     </font>
    </property>
    <property name="styleSheet">
     <string notr="true">	border: 0px solid #000000;
</string>
    </property>
    <property name="text">
     <string/>
    </property>
   </widget>
   <widget class="QLabel" name="label">
    <property name="geometry">
     <rect>
      <x>125</x>
      <y>30</y>
      <width>250</width>
      <height>31</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <pointsize>20</pointsize>
     </font>
    </property>
    <property name="styleSheet">
     <string notr="true">	border: 0px solid #000000;
	color:blue;</string>
    </property>
    <property name="text">
     <string>Username</string>
    </property>
   </widget>
   <widget class="QLabel" name="label_3">
    <property name="geometry">
     <rect>
      <x>50</x>
      <y>70</y>
      <width>21</width>
      <height>25</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <pointsize>20</pointsize>
     </font>
    </property>
    <property name="styleSheet">
     <string notr="true">	border: 0px solid #000000;
</string>
    </property>
    <property name="text">
     <string>1.</string>
    </property>
   </widget>
   <widget class="QLabel" name="usernameLabel2">
    <property name="geometry">
     <rect>
      <x>125</x>
      <y>100</y>
      <width>250</width>
      <height>25</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <pointsize>20</pointsize>
     </font>
    </property>
    <property name="styleSheet">
     <string notr="true">	border: 0px solid #000000;
</string>
    </property>
    <property name="text">
     <string/>
    </property>
   </widget>
   <widget class="QLabel" name="usernameLabel9">
    <property name="geometry">
     <rect>
      <x>125</x>
      <y>310</y>
      <width>250</width>
      <height>25</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <pointsize>20</pointsize>
     </font>
    </property>
    <property name="styleSheet">
     <string notr="true">	border: 0px solid #000000;
</string>
    </property>
    <property name="text">
     <string/>
    </property>
   </widget>
   <widget class="QLabel" name="usernameLabel7">
    <property name="geometry">
     <rect>
      <x>125</x>
      <y>250</y>
      <width>250</width>
      <height>25</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <pointsize>20</pointsize>
     </font>
    </property>
    <property name="styleSheet">
     <string notr="true">	border: 0px solid #000000;
</string>
    </property>
    <property name="text">
     <string/>
    </property>
   </widget>
   <widget class="QLabel" name="usernameLabel10">
    <property name="geometry">
     <rect>
      <x>125</x>
      <y>340</y>
      <width>250</width>
      <height>25</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <pointsize>20</pointsize>
     </font>
    </property>
    <property name="styleSheet">
     <string notr="true">	border: 0px solid #000000;
</string>
    </property>
    <property name="text">
     <string/>
    </property>
   </widget>
   <widget class="QLabel" name="usernameLabel1">
    <property name="geometry">
     <rect>
      <x>125</x>
      <y>70</y>
      <width>250</width>
      <height>25</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <pointsize>20</pointsize>
     </font>
    </property>
    <property name="styleSheet">
     <string notr="true">	border: 0px solid #000000;
</string>
    </property>
    <property name="text">
     <string/>
    </property>
   </widget>
   <widget class="QLabel" name="label_11">
    <property name="geometry">
     <rect>
      <x>50</x>
      <y>280</y>
      <width>21</width>
      <height>25</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <pointsize>20</pointsize>
     </font>
    </property>
    <property name="styleSheet">
     <string notr="true">	border: 0px solid #000000;
</string>
    </property>
    <property name="text">
     <string>8.</string>
    </property>
   </widget>
   <widget class="QLabel" name="label_8">
    <property name="geometry">
     <rect>
      <x>50</x>
      <y>220</y>
      <width>21</width>
      <height>25</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <pointsize>20</pointsize>
     </font>
    </property>
    <property name="styleSheet">
     <string notr="true">	border: 0px solid #000000;
</string>
    </property>
    <property name="text">
     <string>6.</string>
    </property>
   </widget>
   <widget class="QLabel" name="scoreLabel6">
    <property name="geometry">
     <rect>
      <x>450</x>
      <y>220</y>
      <width>175</width>
      <height>25</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <pointsize>20</pointsize>
     </font>
    </property>
    <property name="styleSheet">
     <string notr="true">	border: 0px solid #000000;
</string>
    </property>
    <property name="text">
     <string/>
    </property>
   </widget>
   <widget class="QLabel" name="label_5">
    <property name="geometry">
     <rect>
      <x>50</x>
      <y>130</y>
      <width>21</width>
      <height>25</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <pointsize>20</pointsize>
     </font>
    </property>
    <property name="styleSheet">
     <string notr="true">	border: 0px solid #000000;
</string>
    </property>
    <property name="text">
     <string>3.</string>
    </property>
   </widget>
   <widget class="QLabel" name="label_12">
    <property name="geometry">
     <rect>
      <x>50</x>
      <y>340</y>
      <width>31</width>
      <height>25</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <pointsize>20</pointsize>
     </font>
    </property>
    <property name="styleSheet">
     <string notr="true">	border: 0px solid #000000;
</string>
    </property>
    <property name="text">
     <string>10.</string>
    </property>
   </widget>
   <widget class="QLabel" name="usernameLabel5">
    <property name="geometry">
     <rect>
      <x>125</x>
      <y>190</y>
      <width>250</width>
      <height>25</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <pointsize>20</pointsize>
     </font>
    </property>
    <property name="styleSheet">
     <string notr="true">	border: 0px solid #000000;
</string>
    </property>
    <property name="text">
     <string/>
    </property>
   </widget>
   <widget class="QLabel" name="label_13">
    <property name="geometry">
     <rect>
      <x>40</x>
      <y>30</y>
      <width>51</width>
      <height>31</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <pointsize>20</pointsize>
     </font>
    </property>
    <property name="styleSheet">
     <string notr="true">	border: 0px solid #000000;
	color:blue;</string>
    </property>
    <property name="text">
     <string>Rank</string>
    </property>
   </widget>
   <widget class="QLabel" name="scoreLabel3">
    <property name="geometry">
     <rect>
      <x>450</x>
      <y>130</y>
      <width>175</width>
      <height>25</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <pointsize>20</pointsize>
     </font>
    </property>
    <property name="styleSheet">
     <string notr="true">	border: 0px solid #000000;
</string>
    </property>
    <property name="text">
     <string/>
    </property>
   </widget>
   <widget class="QLabel" name="scoreLabel8">
    <property name="geometry">
     <rect>
      <x>450</x>
      <y>280</y>
      <width>175</width>
      <height>25</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <pointsize>20</pointsize>
     </font>
    </property>
    <property name="styleSheet">
     <string notr="true">	border: 0px solid #000000;
</string>
    </property>
    <property name="text">
     <string/>
    </property>
   </widget>
   <widget class="QLabel" name="scoreLabel1">
    <property name="geometry">
     <rect>
      <x>450</x>
      <y>70</y>
      <width>175</width>
      <height>25</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <pointsize>20</pointsize>
     </font>
    </property>
    <property name="styleSheet">
     <string notr="true">	border: 0px solid #000000;
</string>
    </property>
    <property name="text">
     <string/>
    </property>
   </widget>
   <widget class="QLabel" name="label_10">
    <property name="geometry">
     <rect>
      <x>50</x>
      <y>310</y>
      <width>21</width>
      <height>25</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <pointsize>20</pointsize>
     </font>
    </property>
    <property name="styleSheet">
     <string notr="true">	border: 0px solid #000000;
</string>
    </property>
    <property name="text">
     <string>9.</string>
    </property>
   </widget>
   <widget class="QLabel" name="label_7">
    <property name="geometry">
     <rect>
      <x>50</x>
      <y>190</y>
      <width>21</width>
      <height>25</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <pointsize>20</pointsize>
     </font>
    </property>
    <property name="styleSheet">
     <string notr="true">	border: 0px solid #000000;
</string>
    </property>
    <property name="text">
     <string>5.</string>
    </property>
   </widget>
  </widget>
  <widget class="QLabel" name="label_34">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>70</y>
     <width>1001</width>
     <height>111</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <family>DIN Condensed</family>
     <pointsize>100</pointsize>
    </font>
   </property>
   <property name="styleSheet">
    <string notr="true">color:red;
     background-color:rgba(255, 255, 255, 0);
</string>
   </property>
   <property name="text">
    <string>camPAIN Leaders</string>
   </property>
   <property name="alignment">
    <set>Qt::AlignCenter</set>
   </property>
  </widget>
  <widget class="QLabel" name="samLabel">
   <property name="geometry">
    <rect>
     <x>780</x>
     <y>20</y>
     <width>201</width>
     <height>191</height>
    </rect>
   </property>
   <property name="text">
    <string/>
   </property>
   <property name="scaledContents">
    <bool>true</bool>
   </property>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
